test_bot/config.py

```diff
@@ -0,0 +1,31 @@
+import logging
+import os
+
+try:
+    import dotenv
+except ModuleNotFoundError:
+    print("Not loading .env")
+else:
+    dotenv.load_dotenv()
+
+
+logging.basicConfig(
+    format="%(asctime)s: [%(levelname)s] (%(threadName)s) %(name)s: %(message)s",
+    level=logging.INFO,
+)
+
+
+class Client:
```

Comment:
Maybe call this `Config` and use `from .config import Config` in other files instead? Would make things a bit shorter (`config.Client.debug` vs `Config.debug`)

Reply:
https://github.com/DisnakeDev/disnake/pull/247/commits/e79ecc62d3b5a0cf290acdf58ac846c33858d304